xbmc/cores/VideoPlayer/DVDSubtitles/SubtitlesAdapter.h

```diff
@@ -75,6 +77,13 @@ class CSubtitlesAdapter
 
   CDVDOverlay* CreateOverlay();
 
+  /*!
+  * \brief Post processing of subtitle, will be called before processing
+  *        AddSubtitle method
+  * \param text The subtitle text
+  */
+  virtual void PostProcess(std::string& text){};
```

Comment:
Since `PostProcess` is called in the scope of the public `AddSubtitle`, it can be made protected.

For others trying to understand this, this method is abstract with no implementation because in the future we might want to have common cleanup/post-processing logic that is applied to all subtitles. At the moment only a few derived classes (codecs and parsers) implement such logic and we don't want all of them to override the "do nothing" behaviour (hence not a pure virtual method). Not sure if that's the best way to design this though, seems to be perfectly valid in cpp

Reply:
done